docs/mining/how-to-stake.md

```diff
@@ -6,7 +6,14 @@ Last updated for v1.1.3
 
 ## How to Stake
 
-Participating in proof-of-stake requires a Decred wallet to be running 24/7. The wallet needs to be always online so that it can be called to vote - if the wallet is unavailable then the votes will be missed and no block reward will be received.
+Staking in Decred is two part process:
```

Comment:
This sentence is missing the indefinite article ( needs letter "a" ):

> Staking in Decred is a two part process

Also instead of "two part", maybe calling it a "two stage" or "two step" process is more fitting.

Reply:
I agree. Do you mind changing this?